src/pages/IDO/IDOPage.tsx

```diff
@@ -0,0 +1,294 @@
+import React, { useState } from 'react'
+import { AutoColumn } from '../../components/Column'
+import styled from 'styled-components'
+
+import { RouteComponentProps } from 'react-router-dom'
+//import { TYPE, StyledInternalLink, ExternalLink } from '../../theme'
```

Comment:
remove unnecessary code and comments from this PR

Reply:
Removed all commented code and code no longer used after edits